main.js

```diff
@@ -0,0 +1,104 @@
+let filter = document.querySelector('.filter');
+let div = document.createElement('div');
+let p = document.createElement('p');
+let select = document.createElement('select');
+
+div.className = 'age-selection';
+div.id = 'age-selection';
+filter.append(div);
+p.className = 'filter-subtitle'
+p.innerHTML = 'What is the age of the child you’re travelling with?';
+select.className = 'children-years-old';
+
+
+
+document.querySelector('.col--guests').addEventListener('click', function (){
+    document.getElementById("form-filter").style.display = 'flex';
+})
+
+let adults = 0;
+let children = 0;
+let room = 0;
+let countAdults = document.querySelector('.count-adults');
+let countChildren = document.querySelector('.count-children');
+let countRoom = document.querySelector('.count-rooms');
+
+document.querySelector('.btn-augment-adults').addEventListener('click', function () {
+    let btn = document.querySelector('.btn-augment-adults');
+    if (countAdults.innerHTML < 30) {
+        adults++
+        countAdults.innerHTML = adults.toString();
+        document.querySelector('.col--guests').value = adults + ' Adults ' + children + ' Children ' + room + ' Room';
+    } else {
+        btn.setAttribute('disable','disable');
+    }
+    btn.setAttribute('disable', 'false')
+})
+
+document.querySelector('.btn-subtract-adults').addEventListener('click', function () {
+    let btn = document.querySelector('.btn-subtract-adults')
+    if (countAdults.innerHTML > 0) {
+        adults--
+        countAdults.innerHTML = adults.toString();
+        document.querySelector('.col--guests').value = adults + ' Adults ' + children + ' Children ' + room + ' Room';
+    } else {
+        btn.setAttribute('disable','disable');
+    }
+    btn.setAttribute('disable', 'false')
+})
+
+document.querySelector('.btn-augment-children').addEventListener('click', function () {
+    let btn = document.querySelector('.btn-augment-children')
+    if (countChildren.innerHTML < 10) {
+        children++
+        countChildren.innerHTML = children.toString();
+        document.querySelector('.col--guests').value = adults + ' Adults ' + children + ' Children ' + room + ' Room';
+    } else {
+        btn.setAttribute('disable','disable');
+    }
+    btn.setAttribute('disable', 'false')
+    div.append(p);
+    div.appendChild(select);
+    for (let i = 1; i < 18; i++) {
+        let option = document.createElement('option');
+        option.innerHTML = i + ' years old'
+        select.appendChild(option);
+    }
+})
```

Comment:
you paste this into 1 select instead of creating others 
![image](https://user-images.githubusercontent.com/20189313/140670393-d5238a05-b4ae-40fb-ac3a-0ec03a32d30a.png)